transforms/ember-object/__testfixtures__/decorators.output.js

```diff
@@ -1,4 +1,4 @@
-import { sum as add, alias } from "@ember-decorators/object/computed";
+import { sum as add, oneWay as noWay, reads, readOnly, alias } from "@ember-decorators/object/computed";
```

Comment:
The imports here are off, it would have to be:

```js
import { sum as add, overridableReads, readOnly, alias } from "@ember-decorators/object/computed";
```

The tricky thing here is the rename for `oneWay`, since we can't import `overridableReads` twice. You could add:

```js
const noWay = overridableReads;
```

potentially, or you could override the rename in this one case.